rules/config

```diff
@@ -82,5 +82,5 @@ DEFAULT_KERNEL_PROCURE_METHOD = build
 # host, then either the appropriate account and group will need to be created on the host
 # manually or changes need to be made when the image is built to create the account and
 # group during installation.
-FRR_USER_UID = 1000
-FRR_USER_GID = 1000
+FRR_USER_UID = 300
```

Comment:
Wondering why are you changing the UID to fall within the system-uid pool? The idea is to allow system-admins to be able to create a dummy user in the host (which will always be higher than uid:1000), that will be utilized within the FRR docker to run routing daemons. Any UID under 1000 is subject to be utilized by the system for admin purposes, and could potentially have elevated rights, which is something that would go against our purpose of running FRR processes with lower capabilities/rights.

Reply:
System IDs are being enforced during package install in the docker in the addgroup call and the pkg install fails, so I had to go under 1000.

Preparing to unpack .../frr_6.0.2-0~sonic.debian8+1_amd64.deb ...
Unpacking frr (6.0.2-0) ...
Setting up frr (6.0.2-0) ...
Adding group `frrvty' (GID 108) ...
Done.
addgroup: The group `frr' already exists and is not a system group. Exiting.
dpkg: error processing package frr (--install):
 subprocess installed post-installation script returned error exit status 1
Processing triggers for systemd (215-17+deb8u8) ...
Errors were encountered while processing:
 frr
Reading package lists...
Building dependency tree...
Reading state information...
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up frr (6.0.2-0) ...
addgroup: The group `frrvty' already exists as a system group. Exiting.
addgroup: The group `frr' already exists and is not a system group. Exiting.
dpkg: error processing package frr (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 frr
E: Sub-process /usr/bin/dpkg returned an error code (1)
The command '/bin/sh -c dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; dpkg_apt /debs/frr_6.0.2-0~sonic.debian8+1_amd64.deb' returned a non-zero code: 1
[  FAIL LOG END  ] [ target/docker-fpm-frr.gz ]